src/group/membership/components/GroupMembers.js

```diff
@@ -109,30 +126,224 @@ const Header = props => {
   );
 };
 
+const RemoveButton = props => {
```

Comment:
suggestion: this is very similar to what's in LandscapeMembers. Can we refactor/reuse a shared component?

Reply:
@paulschreiber I created a common component for the duplicated code in the members pages